src/components/augmentedReality/ARModal.js

```diff
@@ -47,6 +47,7 @@ export const ARModal = ({
   return (
     <Modal
       isListView={isListView}
+      height={isListView && '70%'}
```

Comment:
we talked about 80% and i think it would be good to have a bit more space, as we  will have many items in the list. maybe 85% also works out?

Reply:
Yeah, you're right. I tried 85% and it looks good. I'm updating to 85%.
![B1F85860-71D7-45D0-9130-05F520E2D4A0](https://user-images.githubusercontent.com/11755668/208854876-01acfcc6-991f-49f7-9e8b-4547cb429eb0.png)